app/src/lib/stats/stats-store.ts

```diff
@@ -579,10 +581,12 @@ export class StatsStore {
 
     this.optOut = optOut
 
+    const previousValue = getBoolean(StatsOptOutKey) || null
```

Comment:
This will cause `previousValue` to be null both when there's no stored value and when the stored value is 'false' which seems odd to me. Should this be
```suggestion
    const storedValue = getBoolean(StatsOptOutKey)
    const previousValue = storedValue === undefined ? null : storedValue
```

Reply:
👍 I'll figure out a way to push the type coercion closer to the payload

Reply:
Should be addressed in 63679a5d82ab97260288c1d22fe385e59a0ff177